### PR TITLE
aws-configure: add japanese translation 

### PR DESCRIPTION
**Pull request by uh-zz**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ja/common/aws-configure.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+> AWS CLI の設定を管理します。
+> 詳しくはこちら: <https://docs.aws.amazon.com/cli/latest/reference/configure/>
+
+- 対話形式で AWS CLI を設定する（新しい設定の作成、または既定の更新）:
+
+`aws configure`
+
+- 対話形式で AWS CLI の名前付きプロファイルを設定する（新規プロファイルの作成、または既存プロファイルの更新）:
+
+`aws configure --profile {{profile_name}}`
```
Comment (EmilySeville7cfg) [inline pages.ja/common/aws-configure.md:12]:
```suggestion
`aws configure --profile {{プロファイル名}}`
```

Translation is missing for this placeholder. Google translation suggestion maybe have to be applied.

File: pages.ja/common/aws-configure.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- 特定の設定変数の値を表示する:
+
+`aws configure get {{name}}`
```
Comment (EmilySeville7cfg) [inline pages.ja/common/aws-configure.md:16]:
```suggestion
`aws configure get {{名前}}`
```

Translation is missing for this placeholder. Google translation suggestion maybe have to be applied.

File: pages.ja/common/aws-configure.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws configure set {{name}} {{value}} --profile {{profile_name}}`
+
+- 設定エントリを一覧表示する:
+
+`aws configure list`
+
+- 特定プロファイル内の設定エントリを一覧表示する:
+
+`aws configure list --profile {{profile_name}}`
```
Comment (EmilySeville7cfg) [inline pages.ja/common/aws-configure.md:36]:
```suggestion
`aws configure list --profile {{プロファイル名}}`
```


Translation is missing for this placeholder. Google translation suggestion maybe have to be applied.

File: pages.ja/common/aws-configure.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws configure get {{name}} --profile {{profile_name}}`
+
+- 特定の設定変数の値をセットする:
+
+`aws configure set {{name}} {{value}}`
+
+- 特定プロファイルの設定変数の値をセットする:
+
+`aws configure set {{name}} {{value}} --profile {{profile_name}}`
```
Comment (EmilySeville7cfg) [inline pages.ja/common/aws-configure.md:28]:
```suggestion
`aws configure set {{名前}} {{価値}} --profile {{プロファイル名}}`
```


Translations are missing for these placeholders. Google translation suggestions maybe have to be applied.

File: pages.ja/common/aws-configure.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws configure get {{name}}`
+
+- 特定プロファイルの設定変数の値を表示する:
+
+`aws configure get {{name}} --profile {{profile_name}}`
+
+- 特定の設定変数の値をセットする:
+
+`aws configure set {{name}} {{value}}`
```
Comment (EmilySeville7cfg) [inline pages.ja/common/aws-configure.md:24]:
```suggestion
`aws configure set {{名前}} {{価値}}`
```

Translations are missing for these placeholders. Google translation suggestions maybe have to be applied.

File: pages.ja/common/aws-configure.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws configure --profile {{profile_name}}`
+
+- 特定の設定変数の値を表示する:
+
+`aws configure get {{name}}`
+
+- 特定プロファイルの設定変数の値を表示する:
+
+`aws configure get {{name}} --profile {{profile_name}}`
```
Comment (EmilySeville7cfg) [inline pages.ja/common/aws-configure.md:20]:
```suggestion
`aws configure get {{名前}} --profile {{プロファイル名}}`
```

Translations are missing for these placeholders. Google translation suggestion maybe have to be applied.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
All suggestions made here by myself can be replaced with CI checks.

File: pages.ja/common/aws-configure.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
+# aws configure
+
+> AWS CLI の設定を管理します。
+> 詳しくはこちら: <https://docs.aws.amazon.com/cli/latest/reference/configure/>
```
Comment (kbdharun) [inline pages.ja/common/aws-configure.md:4]:
```suggestion
> 詳しくはこちら: <https://docs.aws.amazon.com/cli/latest/reference/configure/>.
```

**Review by kbdharun — COMMENTED** [review]
The translation LGTM, @uh-zz you need to translate the placeholders / command arguments inside `{{}}`. You can either apply @EmilySeville7cfg's Suggestions from Google translate or you can add in your own. Thanks for your contribution.

File: pages.ja/common/aws-configure.md (hunk 1 above)
Comment (uh-zz) [reply pages.ja/common/aws-configure.md:12]: @EmilySeville7cfg 
Thanks for the review!
It looks good to me, so I'll adopt it.

File: pages.ja/common/aws-configure.md (hunk 2 above)
Comment (uh-zz) [reply pages.ja/common/aws-configure.md:16]: @EmilySeville7cfg
Thanks for the review!
It looks good to me, so I'll adopt it.

File: pages.ja/common/aws-configure.md (hunk 3 above)
Comment (uh-zz) [reply pages.ja/common/aws-configure.md:36]: @EmilySeville7cfg
It same looks good to me, so I'll adopt it.

File: pages.ja/common/aws-configure.md (hunk 4 above)
Comment (uh-zz) [reply pages.ja/common/aws-configure.md:28]: It same looks good to me, so I'll adopt it.

File: pages.ja/common/aws-configure.md (hunk 6 above)
Comment (uh-zz) [reply pages.ja/common/aws-configure.md:20]: It same looks good to me, so I'll adopt it.

File: pages.ja/common/aws-configure.md (hunk 7 above)
Comment (uh-zz) [reply pages.ja/common/aws-configure.md:4]: @kbdharun 
Thanks for the review!
It looks good to me, so I'll adopt it.

File: pages.ja/common/aws-configure.md (hunk 5 above)
Comment (uh-zz) [reply pages.ja/common/aws-configure.md:24]: @EmilySeville7cfg 
Thanks for the review!
I'll just correct the translation of "value".

Comment (uh-zz) [reply pages.ja/common/aws-configure.md]: [7c5a63a](https://github.com/tldr-pages/tldr/pull/9237/commits/7c5a63a52eeb455ac067c3d6de171ebdd8102aa8)

**Comment by uh-zz** [comment]
@kbdharun @EmilySeville7cfg 
Thanks for the review!
I have corrected the part you commented on, so please check again.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by EmilySeville7cfg — APPROVED** [review]

**Pull request merged**